pokemon-showdown

```diff
@@ -16,15 +18,18 @@ try {
 try {
 	require.resolve('sockjs');
 } catch (e) {
-	let command = 'npm install --production';
-	console.log('Installing dependencies: `' + command + '`...');
-	require('child_process').spawnSync('sh', ['-c', command], {stdio: 'inherit'});
+	console.log('Installing dependencies...');
+	spawnSync('npm', ['install', '--production'], {stdio: 'inherit', shell: true});
 }
 
-// start the server
+// Start the server
 
 let port;
 if (process.argv[2]) {
 	port = parseInt(process.argv[2]); // eslint-disable-line radix
 }
-require('./app').listen(port);
+
+// Bootstrap manually to ensure any dependencies that were just installed can
+// be found when running on Windows
+
+require('module')._load('./app', module, false).listen(port);
```

Comment:
I don't think you know what "stable" means.

"Stable" means that it's unlikely to be changed in future versions of Node. Undocumented features usually aren't stable.